src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ServiceUserDTO.kt

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ServiceUserData
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Sex
+import java.util.Date
+
+data class ServiceUserDTO(
+  var pncNumber: String? = null,
+  var crn: String? = null,
+  var nomisNumber: String? = null,
+  var dob: Date? = null,
+  var sex: Sex? = null,
```

Comment:
I saw some discussions about the difference between sex and gender -- which one do we need to capture?

Reply:
see comment on the enum. we use this to be able to determine if the SU needs a male or female specific service.